src/main/kotlin/agallipo/BoardingCard.kt

```diff
@@ -1,59 +1,25 @@
-package agallipo
+    package agallipo
 
-import java.io.File
-import java.io.InputStream
+    import java.io.File
+    import java.io.InputStream
+    //created by alex :)
+    data class BoardingCard(var origin: String?, var destination: String?,  var transport: String?, var gate: String?, var seat: String?)
+    //data class CardInformation(val flight: String?, val gate: String?, val seat: String?)
```

Comment:
Another good practice is to avoid having code commented out without an explanation. Usually, it is better to just delete it if it is useless, but if for some reason you have to leave the code but not functional, add a minimal explanation on why: for example "Temporary disabled due to a performance issue" or whatever other thing.